### PR TITLE
Building With Responsive Design

### PR DESCRIPTION
**Pull request by alexisbec**
In this project we worked with:

- Responsive designs in flexbox and grids

- Media Queries

- Different Font Styles

- Backgrounds

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Goldman&family=Open+Sans&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav class="first-nav">
+        <div>
+            <ul class="left-nav">
+                <li>News</li>
```
Comment (umairarshadbutt) [inline index.html]: Add the navigation using the anchor tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+                <li>Events</li>
```
Comment (umairarshadbutt) [inline index.html]: Add the navigation using the anchor tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+                <li>Business</li>
```
Comment (umairarshadbutt) [inline index.html]: Add the navigation using the anchor tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,592 @@
...
+                <li>AMAs</li>
+                <li>Spaces</li>
+                <li>Terms & Conditions</li>
+            </ul>
+            <ul class="right-nav">
+                <li>About</li>
+                <li>Advertise</li>
+                <li>Jobs</li>
+                <li>Contact</li>
+            </ul>
```
Comment (umairarshadbutt) [inline index.html]: Add the navigation using the anchor tag.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,592 @@
...
+            <li>PLUGGED</li>
+            <li>README</li>
+            <li>GROWTH QUARTERS</li>
+            <li>SHIFT</li>
+            <li>NEURAL</li>
+            <li class="mobile-nav">
+                <span class="search"><i class="fas fa-search"></i></span>
+                <span class="hamburger"><i class="fas fa-bars"></i></span>
+            </li>
+        </ul>
```
Comment (umairarshadbutt) [inline index.html]: Add the navigation using the anchor tag.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,592 @@
...
+        </div>
+    </section>
+    <div class="body-margin">
+        <section>
+            <h2 class="latest news">Latest News ></h2>
+
+            <div class="latest-grid">
+                <div class="elmt">
+                    <img src="./img/latest1.jpeg" alt="latest news" />
+                    <div>
```
Comment (umairarshadbutt) [inline index.html:104]: Use the article tag here

File: index.html [hunk 7]
```diff
@@ -0,0 +1,592 @@
...
+                        <p class="elmt-text">Cloud engineers are making close to $150K a year, here's how you can train to become one</p>
+                        <p class="elmt-details">TNW Deals . 5 hours ago</p>
+                    </div>  
+                </div>
+                <div class="elmt">
```
Comment (umairarshadbutt) [inline index.html]: Use the article tag here

File: index.html [hunk 8]
```diff
@@ -0,0 +1,592 @@
...
+                    <img src="./img/latest2.jpg" alt="latest news" />
+                    <div>
```
Comment (umairarshadbutt) [inline index.html:111]: Use the article tag here. Update it on all page

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @alexisbec and @pacyL2K19 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/99320222-f2dc7a00-288c-11eb-90ee-169d8f263240.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/) ([screenshot](https://user-images.githubusercontent.com/29689273/94326203-fa358480-ffbb-11ea-81ec-b2704fb93212.png) from Pathwright)
- [x] Remove SVG code from HTML file. save the SVGs in assets 
- [x] Add the Navigation link in navbar. For more detail, you can see inline comments
- [x] The search icon should be the rightmost side of sub-navigation row. For more detail, you can visit [TNW](https://thenextweb.com/)
- [x] Use `article` tags properly. for more understanding, you can see the inline comments
- [x] Use the navigation links on articles as the original web page is using
- [x] TNW logo is missing from the footer


### Navbar
 - [x] TNW logo is missing in the clone page at medium size screen. Please add the TNW logo at medium size screen.

|[Source Page](https://thenextweb.com/)|[Clone Page](https://alexisbec.github.io/The-Next-Web-Clone/)|
|:---:|:---:|
|source-page|clone|

### Navbar small
- [x] TNW Logo is missing on the clone page please add it
- [x] Search icon and hamburger button should be right in clone page

|[Source Page](https://thenextweb.com/)|[Clone Page](https://alexisbec.github.io/The-Next-Web-Clone/)|
|:---:|:---:|
|source-page|clone|



### Header
- [x] There are extra spacing in the mobile view, You can see in images. Please manage that space issue

|[Source Page](https://thenextweb.com/)|[Clone Page](https://alexisbec.github.io/The-Next-Web-Clone/)|
|:---:|:---:|
|source-page|clone|


### Latest Deals >
- [x] **See all Deals** is missing
- [x] Add the space between two images in mobile screens

|[Source Page](https://thenextweb.com/)|[Clone Page](https://alexisbec.github.io/The-Next-Web-Clone/)|
|:---:|:---:|
|source-page|clone|


**[OPTIONAL]** For the readme file's screenshot you can use [Multi Mockup](https://techsini.com/multi-mockup/index.php) 






##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this project approved. :+1:

- [OPTIONAL]: The footer logo image is broken. Please fix it.

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

Happy coding 🤓

**Pull request merged**